sleap/io/dataset.py

```diff
@@ -629,7 +629,9 @@ def __contains__(self, item) -> bool:
                 return self.find_first(item[0], item[1].tolist()) is not None
         raise ValueError("Item is not an object type contained in labels.")
 
-    def __getitem__(self, key, *args) -> Union[LabeledFrame, List[LabeledFrame]]:
+    def __getitem__(
+        self, key, *args, **kwargs
+    ) -> Union[LabeledFrame, List[LabeledFrame]]:
```

Comment:
[`__getitem__`](https://docs.python.org/3.7/reference/datamodel.html#object.__getitem__) is syntactic sugar for `[]` indexing -- this does not support kwargs and we shouldn't encourage direct usage of this method.

I think if anything, maybe we should move the logic to the `get()` method and just use `__getitem__` as a wrapper for that.